### PR TITLE
Adjust suggestions about sourcing `env` files

### PR DESCRIPTION
**Pull request by rami3l**
Continuation of #3506 to avoid confusion regarding the `.`/`source` command in various POSIX shells.

### CODE REVIEW

**Review by djc — APPROVED** [review]

**Comment by rami3l** [comment]
Waiting for @safinaskar's explicit approval to make sure the PR has correctly address their original concerns.

**Comment by safinaskar** [comment]
Yes, good PR

**Pull request merged**
